packages/subgraph/schema.graphql

```diff
@@ -130,6 +148,20 @@ type OracleTemplate @entity {
     oracles: [Oracle!]! @derivedFrom(field: "template")
 }
 
+type OracleTemplateFeature @entity {
+    id: Bytes!
+    featureId: BigInt!
+    paused: Boolean!
+    templateSet: OracleTemplateSet!
+    allowed: [OracleTemplateFeatureAllowedAccount!]! @derivedFrom(field: "feature")
+}
+
+type OracleTemplateFeatureAllowedAccount @entity {
+    id: Bytes!
+    address: Bytes!
+    feature: OracleTemplateFeature!
+}
+
 type OraclesManager @entity {
```

Comment:
Aren't we missing the `featuresOwner` and `features` for the `OraclesManager` entity?

Reply:
The `featuresOwner` and `features` attributes are actually applied to the template set entities, so `KPITokenTemplateSet` and `OracleTemplateSet`.